ee/codegen/src/project.ts

```diff
@@ -65,10 +66,6 @@ import {
 import { getNodeId } from "src/utils/nodes";
 import { assertUnreachable } from "src/utils/typing";
 
-export class ProjectSerializationError extends Error {
-  isProjectSerializationError = true;
```

Comment:
moved to `errors.ts`. We'll need to update codegen service to check `instanceof ProjectSerializationError`